packages/fxa-content-server/app/styles/modules/_settings-account-recovery.scss

```diff
@@ -147,3 +147,7 @@ button.button-link {
   background: transparent;
   color: $color-blue;
 }
+
+button.generate-key-link{
+  margin-bottom: 5px;
```

Comment:
Our design team got back to me and they'd actually like this to be `16px`. Will you update this here and edit your commit message to include `'account recovery' button` while we're at it?

Will r+ after, thank you!

Reply:
done! :+1: